misk-testing/build.gradle.kts

```diff
@@ -28,6 +28,7 @@ dependencies {
     exclude(group = "junit")
   }
   implementation(Dependencies.javaxInject)
+  implementation(Dependencies.wispTimeTesting)
```

Comment:
As-is, the calling service will get it in the runtime classpath, but not in the compile-time classpath, which IMO is exactly what we want. If we change it to `api` then the calling service will get it in the compile-time classpath transitively, which will make migrating to the wisp version harder because it won't be obvious where the FakeClock is coming from. As-is, the migration will involve (a) change the code to use wisp and (b) swapping out the misk dep for the wisp dep. Either of those in isolation will cause compilation failure, therefore making it easy to detect a botched migration.

Reply:
To be clear, by "botched migration" I mean "the calling service is using the wisp FakeClock, but getting the dependency transitively through the misk wrapper". This would be "botched" because eventually the misk wrapper will be removed, and that would break the calling service.